Perfect number.py

```diff
@@ -0,0 +1,51 @@
+import math
+
+
+def integer_number(msg):
+    while True:
+        try:
+            v = int(input(msg))
+        except ValueError:
+            print("must be integer value")
+            continue
+        else:
+            return v
+            break
+
```

Comment:
Can we do this without using method `interger_number`

Reply:
yes usman bhai, we can.
the purpose of this method is to get only an integer number, if the user will give the other value it will display the error